recipe/meta.yaml

```
@@ -16,16 +16,18 @@ build:
 
 requirements:
   build:
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - cmake
     - make
-  run_constrained:
-    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
+host:
```

```suggestion
  host:
```

Sorry wasn't indented properly.

---

Oh, thanks, I forgot to add the change before git --amend/ -f push